lib/sql-parser/src/base/operation/Select.js

```diff
@@ -22,7 +22,7 @@ class SelectOperation extends BaseSQL {
     pushQueryAndInjectionWhenHaveUnionObject() {
 
         let query = this.queryHandler();
-        this.queryResult.push([this.baseQuery().join(' '), '__$TABLE_NAME$__', query.toSQL].join(' ').trim());
+        this.queryResult.push([this.baseQuery().join(' '), `__$TABLE_NAME$__${this.hasWhereCondition && !this.union ? ` ${this.whereSqlKeyword} __$COLUMN_NAME$__` : ''}`, query.toSQL].join(' ').trim());
```

Comment:
Titile : Solving the problem when there is no union and there is a where object.
Problem : Cannot add where sql keyword to queryRes array